apps/web/src/pages/Applications/ApplicationSkillsPage/ApplicationSkillsPage.tsx

```diff
@@ -110,7 +110,7 @@ export const ApplicationSkills = ({
     followingPageUrl ?? paths.applicationQuestionsIntro(application.id);
 
   const isSkillsExperiencesIncomplete = isIncomplete(
-    application.user,
+    application.user?.experiences?.filter(notEmpty),
```

Comment:
Wait, not really this PR but is this not just the same as the `experiences` passed in as a prop? :thinking: Or am I missing something? I think we could just do
```suggestion
    experiences,
```

Reply:
You seem right to me, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11218/commits/79ff30cec0ecf94fb975c6474d30943552e5cf34